### PR TITLE
Update log-files.md

### PR DESCRIPTION
**Pull request by kellybos**
Update Windows option to get Desktop logs. Get Logs doesn't exist. Need to use option - Collect support files. Also, Desktop logs are useful for troubleshooting authentication or login issues. Collecting the Desktop log manually from %appdata%\Microsoft\Teams is a helpful step.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/log-files.md [hunk 1]
```diff
@@ -136,8 +136,9 @@ Click on the Microsoft Teams icon in your system tray, and select **Get Logs**.
 The files will be available in ~/.config/Microsoft/Microsoft Teams/logs.txt.
   
 To collect logs for Windows:
-Click on the Microsoft Teams icon in your system tray, and select **Get Logs**.
+Click on the Microsoft Teams icon in your system tray, and select **Collect support files**.
 The logs.txt file will be opened in Notepad automatically.    
+When investigating problems logging into Teams you may need to manually collect the Desktop logs. These log files are located at %appdata%\Microsoft\Teams in Windows.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:141]:
```suggestion

When investigating problems signing into Teams you may need to manually collect the desktop logs. These log files are located at %appdata%\Microsoft\Teams in Windows.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kellybos, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/log-files.md [hunk 2]
```diff
@@ -136,9 +136,11 @@ Click on the Microsoft Teams icon in your system tray, and select **Get Logs**.
 The files will be available in ~/.config/Microsoft/Microsoft Teams/logs.txt.
   
 To collect logs for Windows:
-Click on the Microsoft Teams icon in your system tray, and select **Get Logs**.
+Click on the Microsoft Teams icon in your system tray, and select **Collect support files**.
```
Comment (yirkha) [inline Teams/log-files.md:139]:
```suggestion
Right-click (or select and hold) on the Microsoft Teams icon in the taskbar notification area, and select **Collect support files**.
```
1. https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/right-click
2. https://devblogs.microsoft.com/oldnewthing/20030910-00/?p=42583

Comment (kellybos) [reply Teams/log-files.md:139]: Thanks for correctly indicating this is a right-click function. I don't know what select and hold refers to. Users have to right-click for this option.

Comment (yirkha) [reply Teams/log-files.md:139]: That's how a right-click is done on touch-enabled devices 🙂 Just using the inclusive phrasing suggested by the style guide.

Comment (kellybos) [reply Teams/log-files.md:139]: Sounds good! Thanks.

**Pull request merged**